extensions/amp-access-scroll/amp-access-scroll.md

```diff
@@ -3,7 +3,7 @@ $category@: dynamic-content
 formats:
   - websites
 teaser:
-  text: Integrates wth Scroll, a better internet that doesn't rely on ads.
+  text: Integrates with Scroll paywall.
```

Comment:
Scroll isn't really a "paywall" — can we call this "membership"?

Reply:
Done!